qiskit/transpiler/passmanager.py

```diff
@@ -143,19 +147,56 @@ def _normalize_passes(passes):
                 raise TranspilerError('%s is not a pass instance' % pass_.__class__)
         return passes
 
-    def run(self, circuit):
-        """Run all the passes on a QuantumCircuit
+    def run(self, circuit_or_circuits):
```

Comment:
i think this should be `circuits` similar to the signature `transpile()`

Reply:
Done in 9ea5343e